### ISSUE TITLE
General millisecond diff

### ISSUE THREAD

**Issue by DiegoRBaquero** [visionmedia/debug#526]
Hey guys,

Is there any way to get a general non-namespaced millisecond diff?

It would be great to see if anything is blocking the event loop. And which things take long.

**Comment by Qix-** [comment]
Could you expand a bit on what you mean here?

**Comment by DiegoRBaquero** [comment]
Each name space has it's own millisecond diff. So you see +10s inmediately after a +1ms sometimes because that namespace hadn't been called in +10s and there's no way to know how much time it took since last logged debug call, which should be +1ms or something like that.

![image](https://user-images.githubusercontent.com/2006839/41675877-857ea73a-7489-11e8-8f30-8310a3e970ca.png)

if these are subsequent, it should be +10s for first and +1ms for the second.

See https://github.com/visionmedia/debug/pull/528 where I provide a fix.

**Comment by Qix-** [comment]
I agree that this would be useful to certain users but not as a blanket case as namespaced diffs are vitally important to many people.

Check #582 for a potential solution.

**Comment by Qix-** [comment]
Closing in favor of #582.
